# Traceback (most recent call last):
#   File ".\py08_ValueError.py", line 2, in <module>
#     int("안녕하세요")
# ValueError: invalid literal for int() with base 10: "안녕하세요"

